### ISSUE TITLE
Change craftwork

### ISSUE THREAD

**Issue by Akkadror** [WarEmu/WarBugs#19950]
After asking the NPC for a change of craft (by which I mean that the skill related to this craft does not appear in my abilities).

When I ask for a change again, the NPC asks me to unlearn the craft that does not appear in my abilities, but it does not allow me to unlock the situation.
Quiter the game and clear the cache file, does not change anything.

I tried the manipulation with several characters, it's the same thing
This situation has been active since, at least, Saturday, March 19, 2022 1 p.m.

![Akkad_000](https://user-images.githubusercontent.com/101971675/159157781-48896f59-9b5f-43d6-a3d8-6f71a233addc.jpg)
![Akkad_001](https://user-images.githubusercontent.com/101971675/159157798-60947dc4-e6c2-45bd-82e4-82b68305b84c.jpg)
![Akkad_002](https://user-images.githubusercontent.com/101971675/159157807-1426af19-8e20-440d-ab2d-3f73a72310fe.jpg)
![Akkad_003](https://user-images.githubusercontent.com/101971675/159157810-2bafdc38-2cc7-4aae-ae16-4818245ca3b2.jpg)
![Akkad_004](https://user-images.githubusercontent.com/101971675/159157811-ce5a9664-e5b4-4012-b811-b32d2fcc18a8.jpg)

Thank you for your attention.
Sincere greetings

Akkad

**Comment by lilimx** [comment]
Hey,

uhm Im a bit confused. 
What was your trading/crafting skill before?
What skill did you abandon?

**Comment by Akkadror** [comment]
Hello,
Sorry if I wasn't clear enough

My crafting skill was apothecary et butcher, and I wanted to change to salvager et hedge wyzard.

Now I can't take crafts anymore.
And when I try to change again it does the situation explained above.
Yet the game seems considered that I already have salvager and hedge wyzard since he asks me to unlearn them.

**Comment by lilimx** [comment]
Hey,

just be sure....you unlearned both skills first?
You've tried a different trainer aswell? (another capital city for example)

**Comment by Akkadror** [comment]
yes I unlearned both before, and tested at Karak Eight Peaks too
